internal/versions/versions.go

```diff
@@ -18,10 +18,22 @@ import (
 
 	"github.com/edgelesssys/constellation/v2/internal/compatibility"
 	"github.com/edgelesssys/constellation/v2/internal/constants"
+	consemver "github.com/edgelesssys/constellation/v2/internal/semver"
 	"github.com/edgelesssys/constellation/v2/internal/versions/components"
 	"golang.org/x/mod/semver"
 )
 
+// CLIVersion returns the version of this CLI.
+func CLIVersion() consemver.Semver {
```

Comment:
why don't we replace constants.VersionInfo with the new method?

Reply:
Originally didn't to not import semver into constants, keep `CompatibleWithBinary` and to not replace all callsites of constants.VersionInfo. But I guess it's fine to use CLIVersion.